### ISSUE TITLE
Deprecation Warning - FUEL

### ISSUE THREAD

**Issue by Gitoffthelawn** [kashiif/copy-urls-expert#18]
In Firefox 44, Copy URLs Expert results in the following warning in the Browser Console every time Firefox is loaded:

```
DEPRECATION WARNING: FUEL is deprecated, you should use the add-on SDK instead.
You may find more details about this deprecation at: https://developer.mozilla.org/Add-ons/SDK/
resource://app/components/fuelApplication.js 1458 Application
resource://app/components/fuelApplication.js 726 af_ci
chrome://copy-urls-expert/content/copyurlsexpert.js 704 copyUrlsExpert._updateFuelAppData
chrome://copy-urls-expert/content/copyurlsexpert.js 700 copyUrlsExpert._readTemplates
chrome://copy-urls-expert/content/copyurlsexpert.js 127 copyUrlsExpert.init
chrome://copy-urls-expert/content/copyurlsexpert.js 32 copyUrlsExpert.handleLoad/<

```

**Comment by hmijail** [comment]
In Firefox 45 betas, Copy URLs Expert seems to do nothing...

**Comment by Gitoffthelawn** [comment]
This issue pops up as modified 13 hours ago, but I do not see changes or additions.  Do you know what happened?

**Comment by hmijail** [comment]
Sorry, it was me. I wrote here a new comment about breakage in Firefox 45 beta, but then decided it might be better to make it a new, independent issue; so I deleted it.

**Comment by Gitoffthelawn** [comment]
@hmijail No problem at all.  :-)

Thanks for letting me know.  I thought perhaps I was losing my mind.  ;-)

**Comment by kashiif** [comment]
18b1012 Fixes this, released in v5.2.0

**Comment by Gitoffthelawn** [comment]
Thanks for the fix @kashiif
